sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -750,6 +750,22 @@ object DataSourceStrategy
         PushableColumnWithoutNestedColumn(right), _) =>
           Some(new GeneralAggregateFunc("CORR", agg.isDistinct,
             Array(FieldReference.column(left), FieldReference.column(right))))
+        case aggregate.RegrIntercept(PushableColumnWithoutNestedColumn(left),
```

Comment:
Not required, but it'd be nice to find a way to fix the indents here - should not line up with 'case' in the continuation

Reply:
two indents ?